dashboard/public/i18n/en.json

```diff
@@ -25,6 +25,8 @@
     "username": "Username",
     "undefined": "undefined",
     "password": "Password",
+    "confirm-password": "Confirm Password",
+    "confirm-new-password": "Confirm New Password",
```

Comment:
No need to add 2 new locales. The same result can be obtained by using `confirm-password` only/